dev-haskell/glob/metadata.xml

```diff
@@ -5,4 +5,7 @@
 		<email>haskell@gentoo.org</email>
 		<name>Gentoo Haskell</name>
 	</maintainer>
+	<longdescription>
+		A library for globbing: matching patterns against file paths.
```

Comment:
Thanks for the PR! Minor points:

- Commit messages take the format `<category>/<package>: <description>`, e.g. `dev-haskell/glob: add long description to metadata.xml`
- This long description is actually too short to qualify :) they generally need to be more than 150 characters to be allowed to be included. If they are too short, `hackport` will delete them for you. So no need to worry.